types/has-values/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for has-values 2.0
+// Project: https://github.com/jonschlinkert/has-values
+// Definitions by: Ankan Bhattacharya <https://github.com/Ankan002>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function has(value: unknown): boolean;
```

Comment:
this is not allowed, modules should reflect nature of the JS module, here CJS module with export:
https://github.com/jonschlinkert/has-values/blob/master/index.js#L12
The TS conventions and default configuration allows to use CJS exports just like 'export default' exports, not the other way around